src/js/initializers/banner.js

```diff
@@ -0,0 +1,9 @@
+var whenDOMReady = require('../utils/when-dom-ready');
+
+whenDOMReady(function initBanner () {
+
+  // Banner
+  require('../components/banner');
```

Comment:
I didn't catch this earlier, but `banner.js` should really be exporting an initialisation function which this initialiser code calls, similar to what the Accordion, Footer and other components do. (Apart from `header.js`, which we should probably fix as well. Sorry, I didn't think about this properly before!)